endpoint: "http://salad-svc:8080/embed"
image_topic: "/dji/camera/image"
use_compressed: true
keyframe_interval: 1
min_keyframe_dt: 1.0
request_timeout: 8.0
match_threshold: 0.75
match_margin: 0
min_match_index_diff: 10
seq_consistency: 1
topk: 50
max_templates: 10000
jpeg_quality: 85
embedding_topic: "/salad_vpr/embedding"
match_topic: "/salad_vpr/match"

